crates/minimal-oci-spec/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "minimal-oci-spec"
```

Comment:
2 things:

1. It's minimal now, but could expand into something not so minimal.
2. Why not simply have it as a module? `src/spec.rs` for example.